ui/app/controllers/settings/tokens.js

```diff
@@ -45,8 +49,11 @@ export default class Tokens extends Controller {
     this.store.findAll('auth-method');
   }
 
+  /**
+   * @returns {import('@ember/array/mutable').default<import('../../models/auth-method').default>}
+   */
   get authMethods() {
-    return this.store.peekAll('auth-method');
+    return this.model?.authMethods || [];
   }
 
   get hasJWTAuthMethods() {
```

Comment:
`nit`: This should be named `nonJWTAuthMethods`. This returns a list of methods matching the criteria, not a boolean.

Reply:
Hey @ChaiWithJai , is this about `hasJWTAuthMethods` ? If so, the .any() type signature does in fact return a boolean ([docs](https://api.emberjs.com/ember/2.18/classes/EmberArray/methods/mapBy?anchor=any))